ffi/ffi_serialization.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef __LIBFF_FFI_FFI_SERIALIZATION_HPP__
+#define __LIBFF_FFI_FFI_SERIALIZATION_HPP__
+
+#include <libff/algebra/curves/public_params.hpp>
+
+namespace libff
+{
+
+namespace ffi
+{
+
+// FFI buffers are expected to be big-endian formatted, and padded on
+// the left to be the same size as the in-memory bigint
+// representations.  Coefficients of field elements and, and
```

Comment:
field elements and coordinates (there is an extra `and, and`)

Reply:
fixed.  thanks.